src/global/reco/reco.cc

```diff
@@ -65,22 +63,17 @@ void InitPlugin(JApplication *app) {
     app->Add(new JChainFactoryGeneratorT<InclusiveKinematicsSigma_factory>(
             {"MCParticles", "ReconstructedChargedParticles", "ReconstructedChargedParticleAssociations"}, "InclusiveKinematicsSigma"));
 
-    app->Add(new JChainFactoryGeneratorT<GeneratedJets_factory>(
-            {"MCParticles"}, "GeneratedJets"));
-
-    app->Add(new JChainFactoryGeneratorT<ReconstructedJets_factory>(
-            {"ReconstructedParticles"}, "ReconstructedJets"));
-
-    app->Add(new JChainFactoryGeneratorT<ReconstructedElectrons_factory>(
-        {"MCParticles", "ReconstructedChargedParticles", "ReconstructedChargedParticleAssociations",
-        "EcalBarrelScFiClusterAssociations",
-        "EcalEndcapNClusterAssociations",
-        "EcalEndcapPClusterAssociations",
-        "EcalEndcapPInsertClusterAssociations",
-        "EcalLumiSpecClusterAssociations",
-        },
-        "ReconstructedElectrons"
+    app->Add(new JChainMultifactoryGeneratorT<GeneratedJets_factory>(
+             "GeneratedJets",
+            {"MCParticles"},
+            {"GeneratedJets"},
```

Comment:
We don't need to change these factories to mutifactories at all. They need to output jets (currently modeled as `edm4hep::ReconstructedParticles`, we then refer to constituents that are already written in the MCParticles and ReconstructedParticles collections.

Reply:
Out of curiosity: if we wanted to have multiple jet collections (e.g. a collection of track-only jets and then a collection of particle flow jets), would it be ideal to use multifactories?

Reply:
Ahhhh I see. I guess for my edification: are multifactories exclusively for collections that require multiple inputs (e.g. reconstructed particles)?

Reply:
Multifactories are needed for factories that want to produce multiple outputs.

Just to clarify (cc @wdconinc), we are likely to convert all factories (even those producing a single output) to multfactories eventually. Presumably that would not impact the performance, but a consistent interface would simplify implementation of Gaudi-style steering via configuration in Python. Latter is the current informal consensus regarding the direction to go forward. Having said that, producing multiple kinds of jets from the same factories doesn't seem like a sound choice: there is no shared computation between the two procedures, but the shared state opens them up to introducing bugs. We would be much better off making algorithms sufficiently configurable and running those from separately defined factories. In that sense, you might be doing the right thing for the wrong reason.